v_1_1_0/cloudconfig.go

```diff
@@ -1,4 +1,4 @@
-package v2
+package v_1_1_0
```

Comment:
The diff is weird because I changed all versions in the same commit. I try to do better in the future.